src/main/java/de/zalando/aruha/nakadi/controller/EventPublishingController.java

```diff
@@ -50,8 +53,9 @@ public ResponseEntity postEvent(@PathVariable final String eventTypeName, @Reque
         LOG.trace("Received event {} for event type {}", event, eventTypeName);
 
         try {
-            return doWithMetrics(eventTypeName, System.nanoTime(), () -> {
+            return doWithTimerMetric(eventTypeName, System.nanoTime(), () -> {
                 final JSONArray eventsAsJsonObjects = new JSONArray(event);
+                batchSizeHistogram.update(eventsAsJsonObjects.length());
```

Comment:
Yes, I think so. I added it to the ticket ARUHA-156 and gonna implement it in this PR.